testsuite/MDAnalysisTests/core/test_topologyattrs.py

```diff
@@ -121,6 +132,11 @@ class TestAtomids(TestAtomAttr):
     attrclass = tpattrs.Atomids
 
 
+# TODO: unusable because __init__ doesn't take any parameter
+# class TestAtomindices(TestAtomAttr):
```

Comment:
This works for atoms. But `u.atoms.residues.indices` throws a `NotImplementedError`